client/src/index.js

```diff
@@ -1,18 +1,43 @@
+// polyfills
+import 'regenerator-runtime/runtime'
```

Comment:
What do you think about https://babeljs.io/docs/usage/polyfill?

Reply:
can be used - but why bring in something more heavy that we don't need ? a lot what we would polyfill is already provided by the babel transform done by create-react-app